### DISCUSSION TITLE
deepin-screen-recorder的编译依赖问题

### DISCUSSION THREAD

**Discussion by MomentDerek** [linuxdeepin/developer-center#3549] [Development Questions | 开发答疑]
TAG: 5.11.20
设备为擎云L420，海思9006e，arm64
系统为UOS专业版1041
qt版本为5.15

依赖都是通过apt安装的，少什么库就装什么（readme里啥都没）

将`src/utils.cpp`的201行稍作修改以尝试通过编译

```
if (DSysInfo::UosProfessional == DSysInfo::uosEditionType()) {
```

现在卡在这，如果能提供较为完整的编译依赖就更好了

![截图录屏_deepin-terminal_20230117212426](https://user-images.githubusercontent.com/40252940/212910516-757a765a-7dc4-44b4-9b54-0b6733a512cd.png)

**Comment by BLumia** [comment]
> 依赖都是通过apt安装的，少什么库就装什么

可以试下 `apt build-dep` 装依赖，就不用手动装了。

> 现在卡在这

libimagevisualresult 应该是 deepin-image-editor 包提供的，需要先安装这个。

> 如果能提供较为完整的编译依赖就更好了

最近各个项目在向 GitHub 迁移 v23 版的代码，在那之后会督促各个项目更新 README，当然也欢迎直接 PR 完善 :)

顺便，我们也有开发者实时交流群，可以加入 [#deepin-community:deepin.org](https://matrix.to/#/#deepin-community:deepin.org) 进行相关的讨论
